examples/trials/mnist/config_assessor.yml

```diff
@@ -2,7 +2,7 @@ authorName: default
 experimentName: example_mnist
 trialConcurrency: 1
 maxExecDuration: 1h
-maxTrialNum: 1
+maxTrialNum: 100
```

Comment:
a little concern about why would we increase to 100 for the sample config

Reply:
because 1 is too small. assessor would kill unpromising trials early, so it can run more trials. If you think it is too large as an example, is 20 or 30 better?

Reply:
yes, I think that will be better.